cpp/ql/lib/semmle/code/cpp/stmts/Stmt.qll

```diff
@@ -299,6 +319,28 @@ class IfStmt extends ConditionalStmt, @stmt_if {
 class ConstexprIfStmt extends ConditionalStmt, @stmt_constexpr_if {
   override string getAPrimaryQlClass() { result = "ConstexprIfStmt" }
 
+  /**
+   * Gets the initialization statement of this 'constexpr if' statement.
+   *
+   * For example, for
+   * ```
+   * if constexpr (x = y; b) { f(); }
```

Comment:
Same here: Maybe we should have an example with `if constexpr (int x = y; b) { f(); }` or something like that.